modules/bbp-morphology/src/test/resources/invalid/bbp/morphology/fixstainmount/v0.1.0/wrong-type.json

```diff
@@ -1,51 +1,45 @@
 {
   "@context": [
-    "{{base}}/contexts/nexus/core/standards/v0.1.0",
+    "{{base}}/contexts/nexus/shacl/rec-shacl-20170720/v0.1.0",
```

Comment:
The context {{base}}/contexts/nexus/shacl/rec-shacl-20170720/v0.1.0 should only be used for schemas.